Sources/_MatchingEngine/Regex/Parse/LexicalAnalysis.swift

```diff
@@ -942,6 +967,19 @@ extension Source {
     }
   }
 
+  /// Try to consume the start of an absent function.
+  ///
+  ///     AbsentFunctionStart -> '(?~' '|'?
+  ///
+  mutating func lexAbsentFunctionStart(
+  ) -> Located<AST.AbsentFunction.Start>? {
+    recordLoc { src in
+      if src.tryEat(sequence: "(?~|") { return .withPipe }
+      if src.tryEat(sequence: "(?~") { return .withoutPipe }
```

Comment:
Is there a more descriptive name than "with(out) pipe"?

Reply:
I mean the syntax is only used for the repeater syntax, so we could call it `repeater`? Though `withPipe` would then seem out of place, maybe that should be `normal` or something?

Reply:
I think Oniguruma calls them `repeater` and `stopper`, which sounds good enough to me for now.

Reply:
Note this is just for the opening syntax, `(?~|` is not just for absent stoppers, but also for absent expressions and clearers.

Reply:
I've left the naming as-is for now, but am happy to change in a follow-up